layouts/codepen.vue

```diff
@@ -0,0 +1,64 @@
+<template>
+	<BaseLayout layout="codepen" class="p-0">
+		<div class="wrapper">
+			<header v-if="penTitle !== DEFAULT_TITLE">
+				<h1 class="px-14 pt-14">
+					{{ penTitle }}
+				</h1>
+			</header>
+			<iframe
+				:title="penTitle"
+				:src="props.pen"
+				scrolling="no"
+				frameborder="no"
+				loading="eager"
+				allowtransparency="true"
+				allowfullscreen="true"
+			>
+				<slot />
+			</iframe>
+		</div>
+	</BaseLayout>
+</template>
+
+<script setup>
+import { computed } from 'vue';
+import BaseLayout from '../components/BaseLayout.vue';
+
+const DEFAULT_TITLE = 'Untitled';
```

Comment:
Maybe use `null` instead of the `Untitled` string

Reply:
`Untitled` is CodePen's default pen title, so that's why I chose it. I'll make it `null`